docker-for-windows/edge-release-notes.md

```diff
@@ -11,6 +11,41 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.3.2.1
+2020-06-29
+
+> [Download](https://desktop.docker.com/win/edge/46329/Docker%20Desktop%20Installer.exe)
+
+## Security
+
+- Fixed a regression in Docker Desktop Edge 2.3.2.0 where the Docker API was exposed by default on a random port on all interfaces.
```

Comment:
After "2.3.2.0", let's add "on Windows" in case any Mac users read it and worry whether they might be affected too.

Reply:
Fixed.